csharp-basic-topics/ValidateNumberIsPositiveOrNegative/ValidateNumberIsPositiveOrNegativeTests/ValidateNumberTest.cs

```diff
@@ -0,0 +1,76 @@
+using ValidateNumberIsPositiveOrNegative;
+
+namespace ValidateNumberIsPositiveOrNegativeTests
+{
+    [TestFixture]
+    public class ValidateNumberTest
+    {
+        [TestCase(60, 1)]
+        [TestCase(-30, -1)]
+        [TestCase(0, 0)]
+        public void GivenNumber_WhenUsingConditionalMethod_ThenPrintResult(int number, int expectedOutput)
+        {
+            // Arrange - Nothing to arrange in this case
+
+            // Act
+                var actualOutput= NumberValidation.IsPositiveOrNegativeUsingConditionalMethod(number);
+
+            // Assert
+            Assert.That(expectedOutput, Is.EqualTo(actualOutput));
+        }
+        [TestCase(60, 1)]
+        [TestCase(-30, -1)]
+        public void GivenNumber_WhenUsingLeftShiftMethod_ThenPrintResult(int number, int expectedOutput)
```

Comment:
Add test case for 0. Please add in other methods also wherever it is missing.
Also, add one line of space between each method

Reply:
Added test case for 0